<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThongTinNhanVien</class>
 <widget class="QDialog" name="ThongTinNhanVien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thông tin nhân viên</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>641</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#5500ff;&quot;&gt;Thông tin nhân viên&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_NgaySinh">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mã nhân viên</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tên nhân viên</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Địa chỉ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Giới tính</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Số điện thoại</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày sinh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MaNhanVien">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TenNhanVien">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DiaChi">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SoDienThoai">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Nam">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nam</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Nu">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nữ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Them">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sua">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sửa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_XemDanhSach">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xem danh sách</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Xoa">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ThongTinNhanVien">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>496</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_ThongTinNhanVien">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>1301</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
